### DISCUSSION TITLE
model evaluation

### DISCUSSION THREAD

**Discussion by zohregheisary** [Open-Systems-Pharmacology/Forum#1859] [Questions & Problems]
Hi 
How should I fit the data into the model I made? and how to understand how well the model fit/depict the data? Thanks

**Comment by tobiasK2001** [comment]
Hi,

maybe start with a look here: https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/import-edit-observed-data

And than you may find some inspiration here: https://github.com/Open-Systems-Pharmacology/OSP-based-publications-and-content/labels/Tutorial

Best, Tobias
